### ISSUE TITLE
directory.copy is 2x slower in windows

### ISSUE THREAD

**Issue by qdwang** [godotengine/godot#51573]
### Godot version

3.3.2 stable

### System information

windows 10, ubuntu 16.04

### Issue description

`directory.copy` is very slow in windows, much slower than native windows copy, and 2x slower than `directory.copy` in linux

<img width="1129" alt="diff_copy_100m" src="https://user-images.githubusercontent.com/403616/129211257-a74048ac-ccf9-4ef4-a239-412b6b0a4ccc.png">

### Steps to reproduce

```gdscript
extends Node2D

func _ready():
  var dir = Directory.new()
  var t1 = OS.get_system_time_msecs()
  dir.copy("res://dummy.file", "res://dummy.file.copy")
  printt("Time", OS.get_system_time_msecs() - t1)
```

### Minimal reproduction project

[CopySlowIssue.zip](https://github.com/godotengine/godot/files/6976048/CopySlowIssue.zip)

Pls notice there is a 100MB dummy.file inside

**Comment by mhilbrunner** [comment]
>directory.copy is very slow in windows, much slower than native windows copy, and 2x slower than directory.copy in linux

Could you provide how long it approximately takes with your test on each OS?

And the model of your disk may help.

**Comment by Zylann** [comment]
Is filesystem access even comparable between Linux and Windows? I know file access is definitely slower on Windows in general (matters when dealing with lots of files), or has it become worse since a previous version?

**Comment by akien-mga** [comment]
Yeah Linux being twice as fast as Windows for I/O doesn't strike me as a bug, slow I/O is a well-known Windows issue which is not specific to Godot.

More interesting is the claim that Godot copy on Windows would be much slower than the native copy in explorer.exe, in which case there might be room for improvements indeed.

**Comment by Calinou** [comment]
Microsoft generally advises using its proprietary RoboCopy for fast file copies, but this is most likely not something we can do (even using `OS.execute()`).

**Comment by qdwang** [comment]
> Microsoft generally advises using its proprietary RoboCopy for fast file copies, but this is most likely not something we can do (even using `OS.execute()`).

This reminds me that I can use `OS.execute("copy", ....` in windows and `OS.execute("cp", ...` in *nix. I've tested the performance, it's very fast.
Thanks.

**Comment by YYF233333** [comment]
It looks like I can't reproduce this issue on latest master branch. I tried implementing a version of `DirAccess::copy` based on the Windows API, but it was only about 10% faster than the existing implementation (and sometimes showed significant latency fluctuations, whereas the current implementation is more stable). The speed of using the `copy` command directly in the command line is also similar. I guess this issue has been resolved over time, and it should be fine to close it.

**Comment by Calinou** [comment]
Closing per @YYF233333's comment.
